File: Assignment_24/program5.c
///////////////////////////////////////////////////////////////////////
//
// Name :          program5.c
// Discription :   Display String reverse order
// Input :         String
// Output :        String 
// Author :        Shivam Rajendra Kale
// Date :          07/06/2025
//
////////////////////////////////////////////////////////////////////////

/*
  Start
    Accept String from user 
      Display String in reverse order
  Stop    
*/


#include<stdio.h>

void Reverse(char *str)
{
  int iCount = 0,iCnt = 0;

    while(*str != '\0')
    {
        iCount++;
        str++;
    }

    str--;

    for(iCnt = iCount; iCnt > 0; iCnt--,str--)
    {
        printf("%c",*str);
    }

}

int main()
{
   char Arr[20];

   printf("Enter string :\n");
   scanf("%[^'\n']s",Arr);

   Reverse(Arr);

   return 0;

}

File: Assignment_49/program5.java
//////////////////////////////////////////////////////////////////////////
//
// Name :          program5.java
// Discription :   use to Display pattern
// Input :         Integer
// Output :        Charecters
// Author :        Shivam Rajendra Kale
// Date :          14/10/2025
//
//////////////////////////////////////////////////////////////////////////

import java.util.*;

class pattern
{
    public void Display(String str)
    {
        int i = 0, j = 0;
        int Arr[][] = new int [str.length()][str.length()];
        char Brr[] = str.toCharArray();

        // Code
        
        for(i = str.length(); i >= 1; i--)
        {
            for(j = 0; j < str.length(); j++)
            {
                if(i > j)
                {
                    System.out.print(Brr[j]+"\t");
                }
                else 
                {
                    System.out.print("*\t");
                }        
            }
            System.out.println();
        }
    }
}

class program5
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter a name :");
        String str = sobj.nextLine();

        pattern pobj = new pattern();

        pobj.Display(str);

        sobj.close();
    }
}